decentraland/unity-explorer
# fix: After loging out and in again, the music stops playing

## What does this PR change?
Fix #761 

## How to test the changes?
1. Enter the explorer and notice the music is running as usual during the login flow.
2. When you enter the world, check that the music stopped.
3. Open the Explore menu and click in LogOut.
4. Notice the music is running again.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
